build.gradle

```diff
@@ -37,6 +37,8 @@ apply plugin: 'ManifestClasspath'
 apply plugin: 'scalafmt'
 apply plugin: 'com.bmuschko.docker-java-application'
 
+
+
```

Comment:
maybe could keep consistency and have the same number of empty lines between sections

Reply:
delete extra spaces